### ISSUE TITLE
Issue: Discord mp3 links and files not playing

### ISSUE THREAD

**Issue by jameslinimk** [Androz2091/discord-player#284]
Title says it all. Heres the link im using: <https://cdn.discordapp.com/attachments/754773407073042572/812546809183338506/KILLSHOT_Official_Audio.mp3>

**Comment by Snowflake107** [comment]
@jameslinimk Discord attachment links are supported in **develop** version

**Comment by jameslinimk** [comment]
> @jameslinimk Discord attachment links are supported in **develop** version

@Snowflake107 I am using the develop version.

**Comment by jameslinimk** [comment]
Version: <https://github.com/Androz2091/discord-player/tarball/develop>
Code:
```js
message.client.player.play(message, 'https://cdn.discordapp.com/attachments/754773407073042572/812546809183338506/KILLSHOT_Official_Audio.mp3!', true);
```

**Comment by Snowflake107** [comment]
your url `https://cdn.discordapp.com/attachments/754773407073042572/812546809183338506/KILLSHOT_Official_Audio.mp3!` ends with `!` which is invalid

**Comment by jameslinimk** [comment]
> https://cdn.discordapp.com/attachments/754773407073042572/812546809183338506/KILLSHOT_Official_Audio.mp3

@Snowflake107 Just tried without the !, still doesnt work. And opening the link, it is a valid discord attachment

**Comment by Androz2091** [comment]
Hello :wave:
Thank you for reporting this issue. It has been fixed in [v3.3.1](https://github.com/Androz2091/discord-player/releases/tag/v3.3.1).
